internal/api/grpc_definitions_service.go

```diff
@@ -155,14 +155,19 @@ func (s *GrpcDefinitionsService) GetIntegrations(ctx context.Context, _ *emptypb
 				ICE:  int32(item.AutoPiPowertrainToTemplateID[coremodels.ICE]),
 				PHEV: int32(item.AutoPiPowertrainToTemplateID[coremodels.PHEV]),
 			},
+			Points: int64(item.Points),
 		})
+
+		if item.ManufacturerTokenID != nil {
+			result.Integrations[len(result.Integrations)-1].ManufacturerTokenId = item.ManufacturerTokenID.Uint64()
```

Comment:
Same criticism here.

Reply:
yeah agree this would be good to change for more readable version